src/pat/tooltip/tooltip.js

```diff
@@ -26,7 +26,7 @@ define([
     parser.addArgument("height", "auto", ["auto", "max"]);
     parser.addArgument("trigger", "click", ["click", "hover"]);
     parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
-    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
+    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
```

Comment:
Apart from that I still wonder what the advantage is of an extra value to this property. 

> Op 10 sep. 2015, om 12:44 heeft Alexander Pilz <notifications@github.com> het volgende geschreven:
> 
> In src/pat/tooltip/tooltip.js:
> 
> > @@ -26,7 +26,7 @@ define([
> >      parser.addArgument("height", "auto", ["auto", "max"]);
> >      parser.addArgument("trigger", "click", ["click", "hover"]);
> >      parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
> > -    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
> > +    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
> 
> @garbas You are changing content to content_html, doesn't that break any existing use of that argument? Should it be ["auto", "ajax", "content", "content-html", "title"] instead?
> 
> —
> Reply to this email directly or view it on GitHub.
>